Tests/IntegrationTests/Card Tests/StoredPaymentMethodComponentTests.swift

```diff
@@ -168,6 +168,26 @@ class StoredPaymentMethodComponentTests: XCTestCase {
         XCTAssertEqual(viewController?.message, paymentMethod.defaultDisplayInformation(using: nil).title)
         XCTAssertEqual(viewController?.title, localizedString(.dropInStoredTitle, nil, paymentMethod.name))
     }
+    
+    func testStoredPayTo() throws {
```

Comment:
Please make the test name descriptive, `test_subject_action_expectation()` format would make it much easier to understand.